hr_employee_contract_history/README.rst

```diff
@@ -0,0 +1,62 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+=========================
+Employee Contract History
+=========================
+
+* Computes first contract of employees
+* Computes the current contract of employees
+* Compute job ID of employee from job on current contract of employee
+
+
+Installation
+============
+
+To install this module, you need to:
+
+* clone the branch 8.0 of the repository https://github.com/OCA/hr
+* add the path to this repository in your configuration (addons-path)
+* update the module list
+* search for "Employee Contract History" in your addons
+* install the module
+
+Configuration
+=============
+
+This module does not require any special configuration apart from enabling it as
+shown above.
+
+Usage
+=====
+None
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/hr/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/hr/issues/new?body=module:%20hr_employee_contract_chronology%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```

Comment:
The module named was changed to hr_employee_contract_history, please fix it in the issue link